metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1304,6 +1331,30 @@ def _get_published_datasource_project_luid(self, ds):
 
         return None
 
+    def _query_published_datasource_for_project_luid(self, ds: dict) -> None:
```

Comment:
Can we pass in `ds[LUID]` rather than `ds` here?

Reply:
will address in [followup PR](https://github.com/datahub-project/datahub/pull/8417)